client/actions/events/api.ts

```diff
@@ -23,7 +22,7 @@ import {
     getTimeZoneOffset,
 } from '../../utils';
 
-import planningApi from '../planning/api';
+import planningApis from '../planning/api';
```

Comment:
would be good to have some more descriptive naming for this vs `planningApi`, it's not obvious what's the difference